kbs/src/token/mod.rs

```diff
@@ -2,21 +2,41 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
+use async_trait::async_trait;
+use ear::EarAttestationTokenVerifier;
 use jwk::JwkAttestationTokenVerifier;
 use kbs_types::TeePubKey;
 use log::debug;
 use serde::Deserialize;
 use serde_json::Value;
+use std::sync::Arc;
+use strum::EnumString;
+use tokio::sync::RwLock;
 
+mod ear;
 mod error;
 pub(crate) mod jwk;
 pub use error::*;
 
 pub const TOKEN_TEE_PUBKEY_PATH_ITA: &str = "/attester_runtime_data/tee-pubkey";
 pub const TOKEN_TEE_PUBKEY_PATH_COCO: &str = "/customized_claims/runtime_data/tee-pubkey";
+pub const TOKEN_TEE_PUBKEY_PATH_EAR: &str =
+    "/submods/cpu/ear.veraison.annotated-evidence/runtime_data_claims/tee-pubkey";
 
-#[derive(Deserialize, Debug, Clone, PartialEq, Default)]
+#[derive(Deserialize, Default, Debug, Clone, EnumString)]
+pub enum AttestationTokenVerifierType {
+    Jwk,
+    #[default]
+    Ear,
```

Comment:
Please see the previous comment of signing token part of AS' EAR token side. Now EAR token generation will result in a same format of JWT that Simple/ITA uses. Thus we do not the extra token verifier kind but they can share a common one.